templates/cosmic-microservices.yaml

```diff
@@ -1,4 +1,8 @@
 {{- $context := . -}}
+{{- range .Values.namespaces -}}
```

Comment:
We're including namespaces templates at this point, but I haven't seen any namespace templates. Are we missing something?